resources/zib/zib-BloodPressure.xml

```diff
@@ -0,0 +1,304 @@
+          <code value="85354-9" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effectiveDateTime" />
+      <sliceName value="effectiveDateTime" />
+      <short value="BloodPressureDateTime" />
+      <definition value="The date and time at which these blood pressure values were measured." />
+      <alias value="BloeddrukDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-bloodpressure-v3.2.1-2020EN" />
+        <map value="NL-CM:12.4.5" />
+        <comment value="BloodPressureDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Comments on the measured blood pressure. Here, an explanation could be given, for example, of circumstances that may have influenced the patient’s blood pressure, such as pain, stress, exertion and sleep/wake cycles." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-bloodpressure-v3.2.1-2020EN" />
+        <map value="NL-CM:12.4.6" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.bodySite">
+      <path value="Observation.bodySite" />
+      <short value="MeasuringLocation" />
+      <definition value="Anatomical location where the blood pressure was measured." />
+      <alias value="MeetLocatie" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.4.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-bloodpressure-v3.2.1-2020EN" />
+        <map value="NL-CM:12.4.10" />
+        <comment value="MeasuringLocation" />
+      </mapping>
+    </element>
+    <element id="Observation.method">
+      <path value="Observation.method" />
+      <short value="MeasuringMethod" />
+      <definition value="The type of method used to measure blood pressure." />
+      <alias value="Meetmethode" />
+      <binding>
+        <strength value="extensible" />
+      </binding>
+      <mapping>
+        <identity value="zib-bloodpressure-v3.2.1-2020EN" />
+        <map value="NL-CM:12.4.7" />
+        <comment value="MeasuringMethod" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="pattern" />
+          <path value="code" />
```

Comment:
code or code.coding? I'm actually not sure.

Reply:
@ArdonToonstra kun jij hier iets over zeggen?

Reply:
De discriminerende waarde (patternCodeableConcept) staat op `.component.code` dus path staat goed op code.

Reply:
Yes, dank je @ArdonToonstra !